python-checks/src/main/java/org/sonar/python/checks/IdentityComparisonWithNewObjectCheck.java

```diff
@@ -46,6 +49,8 @@
 @Rule(key = "S5796")
 public class IdentityComparisonWithNewObjectCheck extends PythonSubscriptionCheck {
   private static final String MESSAGE_IS = "Replace this \"is\" operator with \"==\".";
+  public static final String IS_QUICK_FIX_MESSAGE = "Replace with \"==\".";
+  public static final String IS_NOT_QUICK_FIX_MESSAGE = "Replace with \"!=\".";
```

Comment:
Same as in the other PR, I think we should always avoid the final `.` in quick fix messages (and my bad if there was any in the tickets!)

Reply:
done